packages/components/src/InputText/InputText.rebuilt.tsx

```diff
@@ -132,13 +116,13 @@ export const InputTextSPAR = forwardRef(function InputTextInternal(
       descriptionIdentifier={descriptionIdentifier}
       description={props.description}
       clearable={props.clearable ?? "never"}
-      maxLength={props.maxLength}
```

Comment:
this addition is actually what caused the old `maxLength` to kick in on v2.

also, that means that this wrapper is calling that `useFormFieldWrapperStyles` hook  itself since as mentioned, we only grabbed the static `styles.input` from the hook. so it was getting called twice but for no reason the first time making even stranger.